packages/sdk-codegen/src/python.gen.spec.ts

```diff
@@ -513,6 +680,43 @@ class PermissionType(enum.Enum):
       expect(actual).toEqual(expected)
     })
 
+    it('needs __annotations__', () => {
+      const type = apiTestModel.types.RequiredResponseWithSingleEnum
+      expect(type).toBeDefined()
+      const actual = gen.declareType('', type)
+      const expected = `
+@attr.s(auto_attribs=True, init=False)
+class RequiredResponseWithSingleEnum(model.Model):
+    """
+    Attributes:
+        query_id: Id of query to run
+        result_format: Desired async query result format. Valid values are: "inline_json", "json", "json_detail", "json_fe", "csv", "html", "md", "txt", "xlsx", "gsxml".
+        user:
+        roles: Roles assigned to group
+    """
+    query_id: int
+    result_format: "ResultFormat"
+    user: "UserPublic"
+    roles: Optional[Sequence["Role"]] = None
+    __annotations__ = {
+        "query_id": int,
+        "result_format": ForwardRef("ResultFormat"),
+        "user": ForwardRef("UserPublic"),
```

Comment:
only classes that have these "bare forward ref" types need the `__annotations__` correction

Reply:
I really appreciate all the comments. Very helpful!